components/combobox/stories/combobox.stories.js

```diff
@@ -171,7 +171,7 @@ QuietGroup.parameters = {
 };
 
 /**
- * Comboboxes have a read-only option for when content in the disabled state still needs to be shown. This allows for content to be copied, but not interacted with or changed. A combobox does not have a read-only option if no selection has been made.
+ * Comboboxes have a read-only option for when content in the disabled state still needs to be shown. This allows for content to be copied, but not interacted with or changed. A combobox does not have a read-only option if no selection has been made. To enable this feature, add `isReadOnly` class to the Combobox. Then within the Textfield component add `isReadOnly` class and `readonly` attribute to the `<input>`.
```

Comment:
```suggestion
* Comboboxes have a read-only option for when content in the disabled state still needs to be shown. This allows for content to be copied, but not interacted with or changed. A combobox does not have a read-only option if no selection has been made. To enable this feature, add the `.isReadOnly` class to the combobox. Then within the textfield component, add  the `.isReadOnly` class and `readonly` attribute to the `<input>`.
```

I'm not sure if this blocks approval, just some sentence-case corrections to the components names, and some additional punctuation.